deployment/deploy.yml

```diff
@@ -1,6 +1,7 @@
 ---
 - hosts: all
-  sudo: yes
+  become: yes
+  become_method: sudo
```

Comment:
Good point, I'll add a check.

There's not much of a reason for bumping minimal Ansible version except that I used my system Ansible which was recently upgraded to `2.1` in homebrew and was too lazy to setup a virtualenv to test with `1.x`.